doc/libf3d/PLUGINS.md

```diff
@@ -58,4 +58,9 @@ The list of existing mimetypes can be find [here](https://www.iana.org/assignmen
 
 ## Loading your plugin
 
-The plugin can be loaded using `f3d::engine::loadPlugin("path or name")` api if you are using libf3d, or `--load-plugins="path or name"` option if you are using F3D application.
+The plugin can be loaded using `f3d::engine::loadPlugin("path or name")` api if you are using libf3d, or `--load-plugins="path or name"` option if you are using F3D application. Several plugins can be specified by giving a comma-separated list.  
```

Comment:
this needs to be documented in user doc, not (only) libf3d doc.

Reply:
I'll move the precedence order in the user documentation, it makes more sense.